models/mongo/conversations.go

```diff
@@ -0,0 +1,63 @@
+// Package mongo is an implementation of models.AccountsRepository
+// that persists data on a MongoDB database.
+package mongo
+
+import (
+	"accounts-service/models"
+	"context"
+
+	// "errors"
+
+	// "github.com/google/uuid"
+	// "go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	// "go.mongodb.org/mongo-driver/mongo/options"
+	"go.uber.org/zap"
+)
+
+type tchatsRepository struct {
+	logger *zap.Logger
+	db     *mongo.Database
+	coll   *mongo.Collection
+}
+
+func NewConversationsRepository(db *mongo.Database, logger *zap.Logger) models.ConversationsRepository {
```

Comment:
Keep in mind you're going to need a `ConversationsRepository` to store conversations and a `MessagesRepository`!